source/Playnite/Playnite.csproj

```diff
@@ -445,6 +445,11 @@
       <SubType>Designer</SubType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="Localization\he_IL.xaml">

```

Comment:
Please remove new language file from this PR (change in csproj and the xaml itself). It has to be merged from PR automatically generated by Crowdin, otherwise these changes will be overwritten next time Crowdin sync is made.